docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_RNNT.md

```diff
@@ -0,0 +1,93 @@
+# Convert ONNX* RNNT to the Intermediate Representation {#openvino_docs_MO_DG_prepare_model_convert_model_onnx_specific_Convert_RNNT}
+
+This instruction convers conversion of RNN-t model from [MLCommons](https://github.com/mlcommons) repository. 
+If you already have a downloaded MLCommons repository you can go right away to **Step 5**.
+
+**Step 1**. Clone the MLCommons repository for inference:
+```bash
+git clone https://github.com/mlcommons/inference
+```
+
+**Step 2**. create a folder for storing pre-trained model and onnx files:
+```bash
+mkdir rnnt_for_openvino 
+cd rnnt_for_openvino
+```
+
+**Step 3**. Download pre-trained weights for PyTorch implementation from https://zenodo.org/record/3662521#.YG21DugzZaQ:
+For UNIX* like systems you can use wget
+```bash
+wget https://zenodo.org/record/3662521/files/DistributedDataParallel_1576581068.9962234-epoch-100.pt
+```
+
+**Step 4**. Install required python packages:
+```bash
+pip install torch toml
+```
+
+**Step 5**. Save the following script in the folder created at the **Step 2**:
```

Comment:
Add main purpose of step here (convert pytorch model to onnx?) and then describe details : save script, run etc (possibly as sublist).

Reply:
You are right. This title didn't describe the main purpose of the step 5. Added that. I did some changes regarding export to ONNX as you suggested. So now there is no need to divide this into subpsteps.